### PR TITLE
Add CombGaussLorentzKT

### PR DESCRIPTION
**Pull request by hankwustfc**
Description of work.

Add CombGaussLorentzKT to Mantid fit functions.

To test:

1.Test if the documentation is added
2.Test if the fit function is working
3.Test the function with the unit test file

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by hankwustfc** [comment]
@AnthonyLim23  Could you please check if the latex equations in .rst work well?

**Comment by AnthonyLim23** [comment]
release notes will be #26827

**Comment by AnthonyLim23** [comment]
can you post a link to the equation from a paper or something (for comparison)?

**Comment by AnthonyLim23** [comment]
for this and the other PR's could you please give some example date to fit to?

**Comment by AnthonyLim23** [comment]
I found the equation and the latex looks fine. However, the link in the doc does not work.

**Comment by hankwustfc** [comment]
That is strange, because it is working perfectly well for me.

Do you have access to this pdf?

https://iopscience.iop.org/article/10.1088/1742-6596/17/1/014/pdf

File: Framework/PythonInterface/plugins/functions/CombGaussLorentzKT.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    def init(self):
+        self.declareParameter("A0", 0.5, 'Amplitude')
+        self.declareParameter("Lambda", 0.1, 'Exponential decay rate')
+        self.declareParameter("Sigma", 0.2, 'KuboToyabe decay rate')
+
+    def function1D(self, x):
+        A0 = self.getParameterValue("A0")
+        Lambda = self.getParameterValue("Lambda")
+        Sigma = self.getParameterValue("Sigma")
+        return A0 * ((1./3) + (2./3) * (1-(Sigma * x) ** 2 - Lambda * x) * np.exp(-(Sigma * x) ** 2/2-Lambda * x))
```
Comment (AnthonyLim23) [inline Framework/PythonInterface/plugins/functions/CombGaussLorentzKT.py]:
```suggestion
        return A0 * ((1./3.) + (2./3.) * (1.-(Sigma * x) ** 2 - Lambda * x) * np.exp(-0.5*(Sigma * x) ** 2-Lambda * x))
```

**Review by AnthonyLim23 — APPROVED** [review]

**Comment by DanNixon** [comment]
Is there a reason locally generated plots are being used rather than taking advantage of the `.. plot::` directive (see https://github.com/mantidproject/mantid/blob/master/docs/source/algorithms/ComputeIncoherentDOS-v1.rst for an example)?

**Comment by AnthonyLim23** [comment]
@hankwustfc Dan has a point. Could you have a look into it?

**Comment by hankwustfc** [comment]
Yes. I will have a look.

**Comment by DanNixon** [comment]
Since this was merged before the discussion was resolved I've created an issue to address the documentation here: https://github.com/mantidproject/mantid/issues/26888.

**Pull request merged**